moped-api/users/users.py

```diff
@@ -193,9 +198,13 @@ def user_update_user(id: str, claims: list) -> (Response, int):
             UserAttributes=updated_attributes,
         )
 
+        # Delete the email if it is different
+        if user_email_before_update != json_data["email"]:
```

Comment:
I'm probably overlooking something, but what would happen if a user's email updated but their roles did not? It looks like their claims would be deleted from DynamoDB and not replaced.

Reply:
It's a great point... admittedly I've only tested when roles are provided and it worked well deleting the claims with the old email and creating the new roles with the new email (line 207). The validator requires roles to be provided, so I would expect it to enforce the update to include the new information.